projects/lxmert/configs/gqa/masked.yaml

```diff
@@ -0,0 +1,57 @@
+dataset_config:
```

Comment:
Can we avoid replication of configs among each other? Same thing is replicated multiple times which makes it hard to process what exactly are the exact changes from one config to other. Rely on default configs and includes as much as possible to avoid replication.